P14-C05-hybridize.ipynb

```diff
@@ -6,19 +6,19 @@
    "source": [
     "# Faster, and portable through hybridizing\n",
     "\n",
-    "The tutorials we saw so far adopt the *imperative*, or define-by-run, programming paradigm. This is the way how we write Python programs. Another commonly used programming paradigm by deep learning frameworks is the *symbolic*, or define-then-run, programming. It consists of three steps\n",
+    "The tutorials we saw so far adopt the *imperative*, or define-by-run, programming paradigm. This is how we write Python programs. Another commonly used programming paradigm by deep learning frameworks is the *symbolic*, or define-then-run, programming. It consists of three steps:\n",
     "\n",
     "- define the workloads, such as creating the neural network\n",
-    "- compile the program into a front-end language, e.g. Python, independent format\n",
+    "- compile the program into a front-end language, e.g. Python, using a language-independent format\n",
```

Comment:
The original structure can seem ambiguous since "independent" needs to be closely associated with something. For now, I will accept your suggestion, but please improve the wording to be not ambiguous later.